crates/sui-bridge-indexer/src/migrations/2024-09-05-180103_governance_actions/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE governance_actions
+(
+    id                          BIGSERIAL    PRIMARY KEY,
```

Comment:
the problem with using serial id as the primary key is that we risk writing duplicated entries

Reply:
resolved offline. see https://github.com/MystenLabs/sui/pull/19252#discussion_r1769365692